linear_algebra/src/power_iteration.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+
+def rayleigh_quotient(input_matrix: np.array, vector: np.array) -> float:
+
+    """
+    Rayleigh Quotient.
+    Fine the Rayliegh quotient of matrix and vector. Given an eigenvector,
+    the Rayleigh quotient is the corresponding eigenvalue.
+
+    Input
+    input_matrix: input matrix.
+    Numpy array. np.shape(input_matrix) == (N,N).
+    vector: input vector.
+    Numpy array. np.shape(vector) == (N,) or (N,1)
+    """
+
+    num = np.dot(vector.T, np.dot(input_matrix, vector))
+    den = np.dot(vector.T,vector)
+
+    return num/den
```

Comment:
Can you remove these lines and use this instead?
```python
from rayleigh_quotient import rayleigh_quotient
```

Reply:
Working on this now, thank you!